daemon/oci_linux.go

```diff
@@ -900,6 +900,7 @@ func (daemon *Daemon) createSpec(c *container.Container) (*specs.Spec, error) {
 	s.Process.NoNewPrivileges = c.NoNewPrivileges
 	s.Process.OOMScoreAdj = &c.HostConfig.OomScoreAdj
 	s.Linux.MountLabel = c.MountLabel
```

Comment:
Is `s.Annotations` guaranteed to be empty here?

Even if so, I think the code should be as follows:
```go
for k, v := range c.Config.Labels{
  vv, ok := s.Annotations[k]
  if ok {
    return nil, errors.Errorf("label %q conflicts, old=%q, new=%q", vv, v)
  }
  s.Annotations[k] = v
}
```

Reply:
@AkihiroSuda  Thanks, will do.

Reply:
@AkihiroSuda If a key already exists in `s.Annotations` we could print a warning msg instead of return directly, wdyt?

Reply:
I think ok.

Reply:
Done.